readme.md

```diff
@@ -22,8 +22,8 @@ Hold `Alt` key (`Option` on MacOS) to measure distances in Figma.
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_stars/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_stars/report/html_report/)
+- [DEMO LINK](https://Diana2369.github.io/layout_stars/)
+- [TEST REPORT LINK](https://Diana2369.github.io/layout_stars/report/html_report/)
```

Comment:
Add these links to the PR description 
<img width="938" alt="image" src="https://github.com/user-attachments/assets/b8fa3677-2ede-4eb6-b890-a20912e997f5" />